extended themeselect.sh to change theme select style

cyb3rgh0u1 opened the pull request:
# Pull Request

## Description

Please read these instructions and remove unnecessary text.

- Try to include a summary of the changes and which issue is fixed.
- Also include relevant motivation and context (if applicable).
- List any dependencies that are required for this change. (e.g., packages or other PRs)
- Provide a link if there is an issue related to this pull request. e.g., Fixes # (issue)
- Please add Reviewers, Assignees, Labels, Projects, and Milestones to the PR. (if applicable)

## Type of change

Please put an `x` in the boxes that apply:

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [x] **Documentation update** (non-breaking change; modified files are limited to the documentations)
- [ ] **Technical debt** (a code change that does not fix a bug or add a feature but makes something clearer for devs)
- [ ] **Other** (provide details below)

## Checklist

Please put an `x` in the boxes that apply:

- [x] I have read the [CONTRIBUTING](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My commit message follows the [commit guidelines](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md#git-commit-messages).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added a changelog entry.
- [x] I have added necessary comments/documentation to my code.
- [x] I have added tests to cover my changes.
- [x] I have tested my code locally and it works as expected.
- [x] All new and existing tests passed.

## Screenshots

![250127_23h53m33s_screenshot](https://github.com/user-attachments/assets/410f9619-bcaf-49f3-bcd6-510d45c18579)

![250127_23h55m12s_screenshot](https://github.com/user-attachments/assets/6de89e85-5f48-4efe-93eb-3515f60c08bd)

![250127_23h56m06s_screenshot](https://github.com/user-attachments/assets/5e959e09-a839-48e1-86ba-b66aa8250bc5)

![250127_23h56m36s_screenshot](https://github.com/user-attachments/assets/40455a01-b76c-4344-943c-e6a476b1b05b)



(if appropriate)

## Additional context

Add any other context about the problem here.

cyb3rgh0u1 commented:
i've extended the `themeselect.sh` script to modify the theme selection menu style the same way i did for hyprdots. this time i added arguments with `themeselect.sh -s` to manage the theme style selection, so there’s no need for a separate script, just as you suggested last time @kRHYME7

kRHYME7 commented on Configs/.local/lib/hyde/globalcontrol.sh:
```diff
@@ -21,6 +21,8 @@ export THEMES_DIR="${XDG_DATA_HOME}/themes"
 #legacy hyde envs // should be deprecated
```
As you can see on the top comment this is deprecated, just put this variable in the themeselect.sh

kRHYME7 commented on KEYBINDINGS.md:
```diff
@@ -148,6 +148,7 @@ Multi-language README support
 | <kbd>SUPER</kbd> + <kbd>ALT</kbd> + <kbd>Down</kbd>  | previous waybar layout |
 | <kbd>SUPER</kbd> + <kbd>SHIFT</kbd> + <kbd>R</kbd>   | wallbash mode selector |
 | <kbd>SUPER</kbd> + <kbd>SHIFT</kbd> + <kbd>T</kbd>   | select a theme         |
+| <kbd>SUPER</kbd> + <kbd>SHIFT</kbd> + <kbd>X</kbd>   | select a style for theme select menu  |
```
Don't add the keybinds yet, will add a menu for subtle things like this.

cyb3rgh0u1 commented:
i've deleted the bind from [KEYBINDINGS.md](https://github.com/HyDE-Project/HyDE/pull/172/files/cb67dbbaa84cec52b62b522ab95f2ace2f1bbd0a#diff-46fb96a48ecccc97123aea06bef5ad2508bfe74b90500d7d10330e0229b35acd) as well as  updated themeselect.sh and removed rofi conf & asset env from globalcontrol.sh

cyb3rgh0u1 commented:
is  everything looks okay now ? if there something else need to change or do, just lemme know @kRHYME7

cyb3rgh0u1 commented:
> I will do some modifications to this PR before merge. Thank you @cyb3rgh0u1
> 
> For now let's not clutter the keybinds, there are something more productive than just theming. Will add a menu soon maybe a right-click menu.

okay, i've no problem with this, do as you please :)

kRHYME7 commented:
@cyb3rgh0u1  Thanks again!
